less/home-page.less

```diff
@@ -17,13 +15,36 @@
   .space-images {
     display: flex;
     flex-wrap: wrap;
-    width: 100%;
+
+    @media @phone {
+      display: flex;
+      flex-direction: column;
```

Comment:
For the space images class you do not need the `flex-direction: column` since you are not displaying 3 images but just one

Reply:
also since you already have `display: flex` set on the `.space-images` class you don't need to call it inside the media query